Android/app/src/main/java/io/github/project_travel_mate/ProfileActivity.java

```diff
@@ -232,6 +232,8 @@ protected void onCreate(Bundle savedInstanceState) {
         });
         displayStatus.addTextChangedListener(mCountCharacters);
         changeImage.setOnClickListener(v -> {
+            Intent getIntent = new Intent(Intent.ACTION_GET_CONTENT);
+            getIntent.setType("image/*");
```

Comment:
how is this change fixing the issue? You are not even using `getIntent` anywhere

Reply:
Sorry I'll fix this shortly